wallets/core/src/hub/mod.ts

```diff
@@ -1,5 +1,5 @@
 export { Namespace } from './namespaces/mod.js';
-
+export type { Subscriber, SubscriberCleanUp } from './namespaces/mod.js';
```

Comment:
I would prefer to not extending `hub` exports for 2 reasons:
1. you can export `SubscriberCleanUp` from `/namespaces` and use `@rango-dev/wallet-core/namespace`.
2. Hub will be the entrypoint of our lib which means `@rango-dev/wallets-core` so I intentionally didn't add it to `package.json` exports as `hub`. when legacy removed, the `hub/mod.ts` will the main entrypoint.

If you've encountered  with any limitation or have any feedback on this, feel free to ask.

Reply:
I added  `Subscriber` and `SubscriberCleanUp` to `namespaces/common`, and imported in Clover from `@rango-dev/wallets-core/namespaces/common`.

I also remove `hub` exporting from `package.json`.